modules/ROOT/pages/6.5_Reference_to_publications_and_to_ESPD_Request.adoc

```diff
@@ -66,14 +66,14 @@ _Table 27. Reference to the ESPD Request, expected elements_
 |*cbc:DocumentTypeCode* |Code |1 |The type of document being referenced, expressed as a code. a|
 *Information Requirement*: link:http://wiki.ds.unipi.gr/display/ESPDInt/BIS+41+-+ESPD+V2.1.0#BIS41-ESPDV2.1-tbr92-013[_tbr92-013_].
 
-*Rule*: For the ESDP-EDM it is compulsory use of the value ESPD_REQUEST from the Code List link:{url-tree-tmp}/dist/cl/ods/[_doc-ref-content-type_]. See also the XML example below on how to specify the ESPD Request.
+*Rule*: For the ESDP-EDM it is compulsory use of the value ESPD_REQUEST from the Code List link:{url-tree-tmp}/codelists/gc/docref-content-type.gc[_doc-ref-content-type_]. See also the XML example below on how to specify the ESPD Request.
```

Comment:
codelists/gc/docref-content-type.gc
link does not work

Reply:
Fixed.